docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -114,33 +148,38 @@ Play is being rebuilt and progressively rolled out to more users. This section e
 For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
 Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker distribution for development and `false` by default on the Kubernetes distribution for production use.
-
-Play uses Zeebe 8.2. Any BPMN elements unavailable in Zeebe 8.2, such as signal events, will not be available in Play.
+For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag in Web Modeler. It is `true` by default for the Docker and Kubernetes distributions.
 
 :::note
-[Inbound Connectors](/components/connectors/connector-types.md#inbound-connectors) and [Connectors in hybrid mode](/guides/use-connectors-in-hybrid-mode.md) do not connect to external systems and must be completed manually.
-[Start events with forms](/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md#embed-form-in-start-event) will be completed without the form being shown.
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable. However, they can be inferred from the output variable.
+[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
 :::
 
-There are some bugs related to Play’s architecture. These will be resolved when Play is integrated with Camunda development clusters.
-
-- **Timer events** sometimes fail silently. Try refreshing the page.
-- Deployment sometimes fails with a network error in the **Output** panel of the **Implement** mode. If these errors repeat, let your Play cluster expire and try again.
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
 
-## Alpha feature
+## Configuration for Self-Managed
```

Comment:
There should be a reference to the Helm setup. Where should the configuration be? Id expect more customization here, so examples are less meaningful. A link to existing docs would be ideal so we don't have to maintain it

Reply:
I don't think we have a proper documentation here, as raised in https://camunda.slack.com/archives/C06SWGVGQ84/p1724327056710069

and I am not sure if I can provide one myself, as the release has not yet happened and we don't have Play enabled in helm charts for alpha releases. I don't feel comfortable without testing myself and seeing the setup actually works, before providing something here.

Reply:
> we don't have Play enabled in helm charts for alpha releases

Despite this, do we link to this documentation anywhere in the Self-Managed docs, or is this noted somewhere?

CC @conceptualshark

Reply:
The `PLAY_ENABLED` option for should be added to the 8.6 [web modeler config docs](https://docs.camunda.io/docs/next/self-managed/modeler/web-modeler/configuration/). You should be able to link out to that configuration for both Docker and Helm and not have to maintain any config on this page, including the following alpha feature section. 

Instead, I'd recommend adding the `PLAY_ENABLED` information (and link to the config docs) at the top of this guide, as a part of "Opening Play," so SM users can immediately get to the page they need for this to run.

Reply:
the problem is, https://signup.camunda.com/docker-compose/ page does not contain information about how to run Play. if Web Modeler is started using the instructions in the page, Operate and Tasklist (that was started in the first step `Zeebe, Operate and Tasklist – to automate` , wouldn't be using OAuth, which prevents Play (which uses Operate and Tasklist APIs) to be properly authenticated, therefore it doesn't work. This was the best solution I could find, in order to guarantee there's a working instructions once the docs are out. 

for helm setup , I couldn't find clean instructions to run helm charts with web modeler. although, I can already see  PLAY_ENABLED flag is mentioned in the page you linked (see https://docs.camunda.io/docs/next/self-managed/modeler/web-modeler/configuration/#feature-flags) . only difference is that, feature flag will be true by default.

To be honest, I am not sure how to fix this part of the documentation regarding these conditions. but please feel free to commit some changes if you do find a solution.

regarding this:

> For the scope of this PR, it might be better to remove the Docker/SM instructions, and create a follow-up issue to reorganize this page after these updates are merged. I'd be happy to take that on!

I wanted to at least mention the default configuration values that can be used for the docker setup (that is unfortunately not up-to-date in the link I shared), so users better understand what values to put there

Reply:
@conceptualshark the follow up ticket you mentioned might also be about removing the instructions and using a link instead, when there is one. this way we can keep the instructions here for now. wdyt?